lib/pure/times.nim

```diff
@@ -296,7 +297,8 @@ when defined(nimHasStyleChecks):
   {.pop.}
 
 type
-  MonthdayRange* = range[1..31]
+  MonthdayRange* = range[0..31]
+    ## 0 represents an invalid month
```

Comment:
This should be "0 represents an invalid day of the month"

Reply:
fixed